app/services/analytics_service.rb

```diff
@@ -1,11 +1,11 @@
 class AnalyticsService
   class << self
-    def call(event_type:, page:, assessment_id: nil, browser_id: nil)
+    def call(event_type:, page:, assessment_code: nil, browser_id: nil)
```

Comment:
this service is only ever called with 4 parameters, so we can remove the defaults